packages/next/src/lib/constants.ts

```diff
@@ -118,9 +118,13 @@ const WEBPACK_LAYERS_NAMES = {
    */
   actionBrowser: 'action-browser',
   /**
-   * The layer for the API routes.
+   * The Node.js bundle layer for the API routes.
    */
-  api: 'api',
+  apiNode: 'api-node',
```

Comment:
we need to rename api to api-node to avoid `api` easily matching `api-edge` with webpack rule sets